pkg/scheduler/api/node_info.go

```diff
@@ -121,6 +121,11 @@ func (ni *NodeInfo) AddTask(task *TaskInfo) error {
 	// change will not impact resource in node.
 	ti := task.Clone()
 
+	// add backfill flag if not done intially yet during cloning snapshot
+	if CheckBackfill(ti.Pod) && !ti.IsBackfill {
```

Comment:
I thought you planed to delete this `if` statement.

Reply:
it is. the last update is from before we talked.